src/arguments/item.ts

```diff
@@ -8,7 +8,10 @@ import { stringMatches } from '../lib/util';
 export default class extends Argument {
 	async run(itemName: string) {
 		// guarantee all characters are numbers
-		if (parseInt(itemName).toString().length === itemName.length) {
+		if (
+			!isNaN(parseInt(itemName)) &&
+			parseInt(itemName).toString().length === itemName.length
+		) {
```

Comment:
I made changes to `getOSItem` already which allow it to accept item IDs as strings, maybe makes this unnecessary?

Reply:
this makes sure the passed string is actually a number by checking that when parsed its length is the same, the isNaN catches the case that was found yesterday where you could pass `cod` return `NaN` and pass the length check.  without the length check `3rd` age would be interpreted as a number